screener.config.js

```diff
@@ -0,0 +1,12 @@
+module.exports = {
```

Comment:
@eriklharper can we make this file in typescript?

Reply:
Not sure if that would work?  We would have to compile it out so that `screener-storybook` can parse and read it.